### ISSUE TITLE
country_code isn't working

### ISSUE THREAD

**Issue by nedbaker12** [sumithemmadi/truecallerpy#23]
i m unable to search the number other than the country code number i logged in with. for example if i use US number to login then if i try to search for uk number using country_code. its still taking US country instead of the uk country code. how to fix this?

**Comment by sumithemmadi** [comment]
@nedbaker12 Install latest version of [truecallerpy](https://pypi.org/project/truecallerpy)

and use it like this

```py
import asyncio
from truecallerpy import search_phonenumber

phone_number = "2071234567"
country_code = "GB"
installation_id = "Your installation ID"

response = asyncio.run(search_phonenumber(phone_number, country_code, installation_id))
print(response)
```

**Comment by nedbaker12** [comment]
yes i used it. still its taking the country code of the country i registered with.

**Comment by nedbaker12** [comment]
hello Please this issue. i m using the latest build. same code as example still the issue is the same

**Comment by libasseld** [comment]
Hello @nedbaker12  i have same issue, did you solve it ?

**Comment by nedbaker12** [comment]
> Hello @nedbaker12 i have same issue, did you solve it ?

no its not fixed @sumithemmadi please look into this
